src/components/Signin.jsx

```diff
@@ -26,7 +27,9 @@ export default function Signin() {
           </Form>
         </Card.Body>
       </Card>
-      <div className="w-100 text-center mt-2">Do not have an account? Sign up</div>
+      <Link to="/signup">
```

Comment:
Great job on creating the link that connects the Signup Page and the Signin Page.